### PR TITLE
Include ground_plane and sun models in Gilbreth world.

### PR DESCRIPTION
**Pull request by mbharatheesha**
The ground_plane and sun models are used in the Gilbreth world. As a result, the first attempt to start the simulation fails as `gilbreth_controller_manager` loses patience while waiting for the models to download from gazebosim.org. This patch just adds the two models into `gilbreth_gazebo/models` directory so that there is no download necessary.

### CODE REVIEW

**Comment by jrgnicho** [comment]
@mbharatheesha excellent, I'll test it as soon as I get a chance.

File: gilbreth_gazebo/models/ground_plane/model-1_4.sdf [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        <material>
+          <script>
+            <uri>file://media/materials/scripts/gazebo.material</uri>
+            <name>Gazebo/Grey</name>
+          </script>
+        </material>
+      </visual>
+    </link>
+  </model>
+</sdf>
```
Comment (jrgnicho) [inline gilbreth_gazebo/models/ground_plane/model-1_4.sdf]: why is it necessary to have multiple versions?

File: gilbreth_gazebo/models/sun/model.config [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+  <author>
+    <name>Nate Koenig</name>
+    <email>nate@osrfoundation.org</email>
+  </author>
+
+  <description>
+    A directional light for the sun.
+  </description>
+</model>
```
Comment (jrgnicho) [inline gilbreth_gazebo/models/sun/model.config:15]: Same as above

Comment (mbharatheesha) [reply gilbreth_gazebo/models/sun/model.config:15]: Multiple versions are indeed not necessary. My mistake. I just copied the downloaded files to the corresponding model folders. Didn't pay attention to the duplicate versions. I will remove them and update the branch.

Also, I am not well versed in this area. So, I wonder about the following questions:
1) Is there a reason older versions are also downloaded with the latest versions on the first instance?
2) Including the duplicate models, both `sun` and `ground_plane` were a few kilobytes in size each (~4k). Not sure why it takes so long to download them anyway.

Comment (jrgnicho) [reply gilbreth_gazebo/models/sun/model.config:15]: @mbharatheesha I don't know much about the internals of gazebo's caching code so I wouldn't know why older versions get saved.
Somehow, my machine only  had models for the sun model and not the ground and my sdf files were only ~500 bytes.

**Comment by jrgnicho** [comment]
@mbharatheesha thanks for the contribution.

**Pull request merged**